createrepo_c.spec

```diff
@@ -60,6 +60,7 @@ BuildRequires:  zchunk
 %if %{with libmodulemd}
 BuildRequires:  pkgconfig(modulemd-2.0) >= 2.3.0
 BuildRequires:  libmodulemd
+Requires:       libmodulemd%{?_isa} >= 2.3.0
```

Comment:
We should probably change this to have the version value controlled by a macro, since it needs to be the same at build-time and runtime.

Reply:
Fixed.